lib/algos/list/basic.js

```diff
@@ -25,20 +27,38 @@ class List {
         this.keys = 0;
     }
 
+    genMDParams() {
```

Comment:
genMDParams will allocate an object and remove useless property.
But leveldb accept and ignore the undefined property.
This is not an issue but we should be able to avoid this step.